Dev/Gitty/git_python.5m.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# <bitbar.title>Gitty</bitbar.title>
+# <bitbar.version>v0.1</bitbar.version>
+# <bitbar.author>Timothy Barnard</bitbar.author>
+# <bitbar.author.github>collegboi</bitbar.author.github>
+# <bitbar.image>https://raw.githubusercontent.com/collegboi/Bitbar-Gitty/master/image1.png</bitbar.image>
```

Comment:
This image doesn't exist anymore.  Should either replace it with one that does (recommended) or remove the metadata.